prebid-mobile/pbm-api/ios/code-integration-ios.md

```diff
@@ -141,8 +137,13 @@ If you integrate Prebid Mobile with GMA SDK with version equal or higher than 10
         break
     }            
 }   
+```
+
 {% endcapture %}
-{% capture gma11 %}Prebid.initializeSDK(gadMobileAdsVersion: GADGetStringFromVersionNumber(GADMobileAds.sharedInstance().versionNumber) { status, error in
+{% capture gma11 %}
```

Comment:
The spaces were omitted intentionally to avoid rendering issues.
<img width="886" alt="image" src="https://github.com/user-attachments/assets/70c4ffe9-cdc9-422e-a830-17eaa6869261" />

Reply:
It looks like a conflict here. The linter requires the addition of a line before and after the code block. I'll take a look at how to fix it

Reply:
I suppose removing code block syntax may help